### ISSUE TITLE
video not work in ios

### ISSUE THREAD

**Issue by mohamedanwer123** [itsnubix/react-native-video-controls#200]
video not work in ios but work fine in android 

and git this error : video unavailable 

react native : 63.5

my code : 

<View style={{borderRadius:5,width:isFullScreen?responsive.responsiveWidth(100):responsive.responsiveWidth(90),height:isFullScreen?responsive.responsiveHeight(100):responsive.responsiveHeight(30)}}>
        <VideoPlayer
        onEnterFullscreen={()=>setIsFullScreen(true)}
        onExitFullscreen={()=>setIsFullScreen(false)}
        source={{ uri: 'https://vjs.zencdn.net/v/oceans.mp4' }}
        disableBack	
        repeat
        seekColor='red'
        tapAnywhereToPause
        videoStyle={{borderRadius:5, width:isFullScreen?responsive.responsiveWidth(100):responsive.responsiveWidth(90),height:isFullScreen?responsive.responsiveHeight(100):responsive.responsiveHeight(30)}}
        style={{backgroundColor:'black', borderRadius:5,width:isFullScreen?responsive.responsiveWidth(100):responsive.responsiveWidth(90),height:isFullScreen?responsive.responsiveHeight(100):responsive.responsiveHeight(30),}}
        />
      </View>

**Comment by ram4git** [comment]
Any solution? 
It just started showing all of a sudden

**Comment by GuleriaAshish** [comment]
Yes, am i also facing the same issue, It was working fine fews days back and all of sudden it started showing me video unavailable error.
Did you guys find any solution for this ?

here is the error that i receive in onError method:
{"error": {"code": -11829, "domain": "AVFoundationErrorDomain", "localizedDescription": "Cannot Open", "localizedFailureReason": "This media may be damaged.", "localizedRecoverySuggestion": ""}, "target": 9499}

but that is not the reason , as i have checked by hitting my video url at web browser and it shows me the video without any error.

**Comment by Great-hijack** [comment]
I wasted 3 days for this issue, please help me.

**Comment by dhairyasenjaliya** [comment]
any solution ?

**Comment by achall9** [comment]
Hey any word on this? I am stuck also.

**Comment by jhjdev** [comment]
Setting:

` source={{ type: 'mp4', uri: 'https://vjs.zencdn.net/v/oceans.mp4' }}`

Seems to work. But I'm using React Native 0.68.0. Which might have something to do with it.

**Comment by cihanbas** [comment]
if you are using the encrypt or drm you need to activated the apple fairplay

**Comment by imyuvsinha** [comment]
have resolved this issue by adding `NSAppTransportSecurity` to true in `info.plist`.


<img width="905" alt="image" src="https://github.com/itsnubix/react-native-video-controls/assets/46275964/51d578f8-0bb9-41d5-93b6-9cba9c0ff4dd">
